deployments/helm/nvidia-device-plugin/templates/role-binding.yml

```diff
@@ -14,4 +14,22 @@ roleRef:
   kind: ClusterRole
   name: {{ include "nvidia-device-plugin.fullname" . }}-role
   apiGroup: rbac.authorization.k8s.io
+{{ if eq .Values.microshift "enabled" }}
```

Comment:
@arthur-r-oliveira these look similar to the changes made in https://github.com/NVIDIA/k8s-dra-driver/pull/76/files for the k8s-dra-driver. Is there a similar check to:
```
{{- if .Capabilities.APIVersions.Has "security.openshift.io/v1/SecurityContextConstraints" -}}
```
for microshift that we can use instead of requiring that users opt-in specifically?

Reply:
@elezar thanks for the heads-up! We can use the same option for MicroShift, as we are taking about the same API in this case. I've just moved back values.yaml to previous state and update role.yml and role-binding.yml with ` .Capabilities.APIVersions.Has`.